### ISSUE TITLE
Load community fields incrementally

### ISSUE THREAD

**Issue by NeoVier** [cambiatus/frontend#640]
**What we have**
We load the entire `Community` object at once. Sometimes we already have the community loaded, but need to refresh a certain field, and we need to query for the entire community again.

**Proposal**
Load only some fields initially. More complex/heavy fields, such as `contributions`, `objectives`, etc. could be loaded as they're needed.

**Why**
This could give us faster load times for people with poor internet. Also, if users are on mobile data, it can save them that as well.

**How**
Set these heavy fields as `RemoteData.NotAsked` initially. We can use the `BroadcastMsg` API we created a while ago to "request" new fields. With that, we have endless possibilities. We can reload a single field whenever we want, and we could even experiment with requesting fields before the user actually needs it (e.g. if they're about to click on a button that would take them to a page that needs objectives, we could query for objectives before the user actually clicks the button).

The broadcast API really provides a lot of nice possibilities we could take advantage of.

**Additional context**
It would be nice to have a discussion on pros & cons before starting to implement this! Maybe we don't even need faster load times, or maybe this isn't the way to go about it, or maybe the work put into this isn't really worth it. But maybe it could be really nice for some people in rural areas, or out in the field (using mobile data)

We could do the same thing for pretty much **any** structure we get from GraphQL, but `Community` seems to be the biggest one as of now, and if we go through with this, we could use this example on other structures!

**Comment by NeoVier** [comment]
@lucca65 would love to hear your thoughts about this! It's something I've been thinking about lately, and we should be able to do it with a very nice API and (hopefully) with not so much effort
